src/page-viewer/styles.css

```diff
@@ -0,0 +1,6 @@
+.root {
+    float: right;
+    padding: 4px 4px 4px 12px;
```

Comment:
Looks like we'll either have to pass all the styles down, as passing down the className prop will overwrite the component's own className or do something like this:

```jsx
export const LinkToLocalVersion = ({page, children, ...props}) => {
    let classes = classNames({
        [styles.root]: true,
        [props.className]: true
    })

    return (
        <a
            href={`data:text/html;charset=UTF-8,${pageToHTML(page)}`}
            title="Stored text version available"
            {...props}
            className={classes}
        >
            {children}
        </a>
    )
}
```

What do you reckon @Treora?